app/uk/gov/hmrc/leakdetection/persistence/ReportsRepository.scala

```diff
@@ -101,25 +102,35 @@ class ReportsRepository @Inject()(reactiveMongoComponent: ReactiveMongoComponent
   def getDistinctRepoNames: Future[List[String]] =
     collection
       .distinct[String, List](
-        "repoName",
-        Some(hasUnresolvedErrorsSelector)
-      )
-      .map(_.sorted)
+      key = "repoName",
+      query = Some(hasUnresolvedErrorsSelector),
+      readConcern = ReadConcern.Majority,
+      collation = None).map(_.sorted)
 
   def howManyUnresolved(): Future[Int] =
-    collection.count(Some(Json.obj("inspectionResults" -> Json.obj("$gt" -> JsArray()))))
+    collection.count(
+      selector = Some(Json.obj("inspectionResults" -> Json.obj("$gt" -> JsArray()))),
+      limit = None,
```

Comment:
to check, these are all the defaults for count which would be what was done before?

Reply:
A lot of these are the defaults that I've been able to work out from looking at the calls the deprecated version was making. The exception here being `ReadConcern.Majority`, which I'll comment on below.